### PR TITLE
Fixed Video first resize problem

### PR DESCRIPTION
**Pull request by beaubbe**
THe first resize of the video was done too soon. It was done before it was loaded and had its size. I added a listener for "loadedmetadata" event. When triggered, the first resize is done.

### CODE REVIEW

File: src/com/video.js [hunk 1]
```diff
@@ -127,10 +133,8 @@
 			// attach events
 			o.video.on('timeupdate', onTimeUpdate)
 				.on('canplay', onCanplay)
-				.on('playing', onPlaying);
```
Comment (nitriques) [inline src/com/video.js]: missing a `;`

File: src/com/video.js [hunk 2]
```diff
@@ -14,7 +14,8 @@
 		onTimeUpdate: $.noop,
 		onCanplay: $.noop,
 		onPlaying: $.noop,
-		isVideoResized: true
+		isVideoResized: true,
```
Comment (nitriques) [inline src/com/video.js]: The state should not live in the options... set a class or use `.data()`.

File: src/com/video.js (hunk 1 above)
Comment (beaubbe) [inline src/com/video.js]: There is no missing ;

Comment (nitriques) [inline src/com/video.js]: Yeah you're right

**Pull request merged**